src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,54 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// SpaceX Rockets API link
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+// Initial state for the store
+const initialState = {
+  rocketsList: [],
+  isLoading: false,
+};
+
+// Exportable method to fetch data from the SpaceX API
+export const getRockets = createAsyncThunk('rockets/getRockets', async (name, thunkAPI) => {
+  try {
+    const resp = await axios(url);
+    return resp.data;
+  } catch (error) {
+    return thunkAPI.rejectWithValue('something went wrong');
+  }
```

Comment:
[OPTIONAL]: I think, it's recommended to avoid using `try, catch` statments inside `createAsyncThunk` because by default it can handle 3 states (`pending`, `fulfilled` and `reject`) You `try,catch` becomes innecessary. Take a look at [this example](https://stackoverflow.com/questions/68896099/why-doesnt-my-try-catch-block-handle-dispatch-error) if is necessary

Reply:
Thank you, @anderCM, I will review the documentation provided and make the changes 👌